src/pages/Admin/Venue/TemplateForm.tsx

```diff
@@ -151,9 +155,15 @@ const TemplateCard: React.FC<TemplateCardProps> = ({
 }) => {
   const thumbnailImage = templateThumbImageMap[template];
   const hasThumbnail = !!thumbnailImage;
+  const pickspaceComponentStyles = HIDDEN_BURN_VENUE_TEMPLATES.includes(
+    template
+  )
+    ? { display: "none" }
+    : {};
 
   return (
     <div
+      style={pickspaceComponentStyles}
```

Comment:
can you do that not using inline styles but adding a class name and handling it in css? or alternatively do something like 
```{ canShowTemplate && <div /> }```